### PR TITLE
Fix #34 showrss.info

### LINKED ISSUE #34: Add showrss.info to showRSS company data
Website: https://showrss.info/

"showRSS is a tool that makes it easier for people to track ongoing TV shows. Create a free account, pick your preferred shows and generate a feed link that you can then subscribe to. And because some torrent clients support subscriptions, you can easily automate your setup by just plugging your personal feed."

Screenshot 2023-01-31 at 10 01 13 am

### PR DESCRIPTION
**Pull request by jellizaveta**
https://github.com/AdguardTeam/companiesdb/issues/34

### CODE REVIEW

File: dist/adguard.json [hunk 1]
```diff
@@ -48,6 +48,12 @@
             "url": "https://crashlytics.com/",
             "companyId": null
         },
+        "showrss": {
+            "name": "showRSS",
+            "categoryId": 6,
```
Comment (Alex-302) [inline dist/adguard.json:53]: It does not track users data, just creates feeds with torrent magnet links (if I understand correctly). Maybe better `12`

Comment (jslawler) [reply dist/adguard.json:53]: Yeah, it's a tracker for TV shows, not a tracker of user data.

You select a tv show, add it to your list, then export the rss feed into a third party app such as Catch for macOS to automate tv show downloads.

Comment (jellizaveta) [reply dist/adguard.json:53]: hmmm, then I think it's more of a miscellaneous category - `8`

File: dist/adguard.json [hunk 2]
```diff
@@ -48,6 +48,12 @@
             "url": "https://crashlytics.com/",
             "companyId": null
         },
+        "showrss": {
+            "name": "showRSS",
+            "categoryId": 8,
+            "url": "https://showrss.info/",
+            "companyId": "showRSS"
```
Comment (Alex-302) [inline dist/adguard.json:55]: Not `showrss`?

Comment (jslawler) [reply dist/adguard.json:55]: > Not `showrss` like in https://github.com/AdguardTeam/companiesdb/pull/37/files#diff-58dc3c00cb661de579a63bdf6202d21dc2676f89c5e2faa04149d6421f55fc88R164?

`companyId` should be `showrss` I'd say. `name` should be able to stay as is based off other database entries.

**Review by ameshkov — APPROVED** [review]

**Pull request merged**